### DISCUSSION TITLE
New RQ version 1.14 potentially breaking upgrades/new installs

### DISCUSSION THREAD

**Discussion by AADIP** [netbox-community/netbox#12417] [General]
I have spent the last couple hours debugging why I couldnt get `netbox-rq` to start and found https://github.com/rq/django-rq/issues/589 stating the same thing. It looks like https://github.com/rq/rq/releases/tag/v1.14 was released ~5h ago at the time of writing will need to be fixed. 

**For now, this was my remediation:**
```
source /opt/netbox/venv/bin/activate
pip3 install --force-reinstall -v "rq==1.13"
service netbox-rq start
```

Anyone else can feel free to add comments or suggestions. I just wanted to get a quick PSA out here.

**Comment by jeremystretch** [comment]
Thanks for the report. This has just been captured under #12415.
